docs/tutorials/tokens/erc1155.md

```diff
@@ -19,7 +19,7 @@ nav_order: 3
 ---
 
 ## Previous steps: Install the FireFly CLI
-If you haven't set up the FireFly CLI already, please go back to the previous step and read the guide on how to [Install the FireFly CLI](./firefly_cli.md).
+If you haven't set up the FireFly CLI already, please go back to the Getting Started guide and read the section on how to [Install the FireFly CLI](./firefly_cli.md).
```

Comment:
I think this link is broken (maybe needs to go up one more level?)

Reply:
Yep. Thanks. I copied it from a different level in the tree